runtime/moonbase/src/lib.rs

```diff
@@ -916,18 +937,70 @@ runtime_common::impl_runtime_apis_plus_common! {
 	impl sp_transaction_pool::runtime_api::TaggedTransactionQueue<Block> for Runtime {
 		fn validate_transaction(
 			source: TransactionSource,
-			tx: <Block as BlockT>::Extrinsic,
+			xt: <Block as BlockT>::Extrinsic,
```

Comment:
Is that the convention for extrinsic ? never really thought about it :)

Reply:
Yeah, I've seen `xt` a lot. Kind of funny that it's similar to `tx`. I don't care which one we use, I just changed this one so it matches the other runtimes.